js/everything.js

```diff
@@ -223,6 +226,9 @@ var ZenPlayer = {
                 if (window.sessionStorage) {
                     var currentTime = plyrPlayer.plyr.embed.getCurrentTime();
                     var videoDuration = plyrPlayer.plyr.embed.getDuration();
+                    if (videoDuration === 0) {
```

Comment:
Sure thing. It's a good place to learn smth about code styling. I'll add both to my next commit as well as fix regexp to `/[?&]t=(\d*)/g`.